python/lsst/obs/decam/ingest.py

```diff
@@ -24,15 +24,6 @@
 import lsst.afw.image as afwImage
 from lsst.pipe.tasks.ingest import ParseTask, IngestTask, IngestArgumentParser
 
-def parseExtname(md):
```

Comment:
This (along with some other changes) is part of 62918c3d5. Commit message is:

    Eliminate the deprecated dataId "side" and "ccd" (DM-6585)

Does this mean this is supposed to be a solution to DM-6585, ie that ticket is being closed as part of this work?